examples/with-react-md-typescript/pages/_app.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { ReactElement } from 'react'
+import Head from 'next/head'
+import { AppProps } from 'next/app'
+
+import Layout from '../components/Layout'
+
+import '../styles/app.scss'
+
+export default function App({ Component, pageProps }: AppProps): ReactElement {
+  return (
+    <Layout>
+      <Head>
+        <link
+          key="Roboto"
+          rel="stylesheet"
+          href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap"
+        />
```

Comment:
I'd prefer not to ship a custom font to an example. These are common sources of performance problems.
```suggestion
```

Reply:
Sounds good, should be removed now!